### PR TITLE
v2.0.0b46

### PR DESCRIPTION
**Pull request by mrakitin**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mrakitin** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -40,7 +40,7 @@ requirements:
     - pytz
     - starlette
     - suitcase-mongo >=0.6.0
-    - tiled >=0.1.0b4
```
Comment (gwbischof) [inline recipe/meta.yaml:43]: Looking at the databroker requirements: https://github.com/bluesky/databroker/tree/v2.0.0b46
I think it only requires tiled >- 0.1.0b4

**Comment by tacaswell** [comment]
> json-merge-patch 0.2 is not supported on this platform

that is the error that is failing the build.

**Comment by mrakitin** [comment]
@conda-forge-admin, please rerender

**Pull request merged**